x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/details/execution_log_table/execution_log_columns.tsx

```diff
@@ -0,0 +1,171 @@
+    name: (
+      <TableHeaderTooltipCell
+        title={i18n.COLUMN_STATUS}
+        tooltipContent={i18n.COLUMN_STATUS_TOOLTIP}
+      />
+    ),
+    field: 'security_status',
+    render: (value: RuleExecutionStatus, data) =>
+      value ? (
+        <EuiHealth color={getStatusColor(value)}>{capitalize(value)}</EuiHealth>
+      ) : (
+        getEmptyTagValue()
+      ),
+    sortable: false,
+    truncateText: false,
+    width: '10%',
+  },
+  {
+    field: 'timestamp',
+    name: (
+      <TableHeaderTooltipCell
+        title={i18n.COLUMN_TIMESTAMP}
+        tooltipContent={i18n.COLUMN_TIMESTAMP_TOOLTIP}
+      />
+    ),
+    render: (value: string) => <FormattedDate value={value} fieldName="date" />,
+    sortable: true,
+    truncateText: false,
+    width: '15%',
+  },
+  {
+    field: 'duration_ms',
+    name: (
+      <TableHeaderTooltipCell
+        title={i18n.COLUMN_DURATION}
+        tooltipContent={i18n.COLUMN_DURATION_TOOLTIP}
+      />
+    ),
+    render: (value: number) => (
+      <>{value ? <RuleDurationFormat duration={value} /> : getEmptyValue()}</>
+    ),
+    sortable: true,
+    truncateText: false,
+    width: '10%',
+  },
+  {
+    field: 'security_message',
+    name: (
+      <TableHeaderTooltipCell
+        title={i18n.COLUMN_MESSAGE}
+        tooltipContent={i18n.COLUMN_MESSAGE_TOOLTIP}
+      />
+    ),
+    render: (value: string) => <>{value}</>,
+    sortable: false,
+    truncateText: false,
+    width: '35%',
+  },
+];
+
+export const GET_EXECUTION_LOG_METRICS_COLUMNS = (
+  docLinks: DocLinksStart
+): Array<EuiBasicTableColumn<AggregateRuleExecutionEvent>> => [
+  {
+    field: 'gap_duration_ms',
+    name: (
+      <TableHeaderTooltipCell
+        title={i18n.COLUMN_GAP_DURATION}
+        customTooltip={
+          <div style={{ maxWidth: '20px' }}>
+            <PopoverTooltip columnName={i18n.COLUMN_GAP_DURATION}>
+              <EuiText size={'s'} style={{ width: 350 }}>
+                <p>
+                  <FormattedMessage
+                    defaultMessage="Duration of gap in Rule execution (hh:mm:ss:SSS). Adjust Rule look-back or {seeDocs} for mitigating gaps."
+                    id="xpack.securitySolution.detectionEngine.ruleDetails.ruleExecutionLog.gapDurationColumnTooltip"
+                    values={{
+                      seeDocs: (
+                        <EuiLink href={`${docLinks.links.siem.troubleshootGaps}`} target="_blank">
+                          {i18n.COLUMN_GAP_TOOLTIP_SEE_DOCUMENTATION}
+                        </EuiLink>
+                      ),
+                    }}
+                  />
+                </p>
+              </EuiText>
+            </PopoverTooltip>
+          </div>
+        }
+      />
+    ),
+    render: (value: number) => (
+      <>{value ? <RuleDurationFormat duration={value} isMillis={true} /> : getEmptyValue()}</>
+    ),
+    sortable: true,
+    truncateText: false,
+    width: '10%',
+  },
+  {
+    field: 'indexing_duration_ms',
```

Comment:
Verify if translated from nanos, should be using `isMillis={true}`?

Reply:
Resolved in https://github.com/elastic/kibana/pull/130072, `isMillis` wasn't the accurate label (they're all millis but gap_duration which is seconds), so moved to `isSeconds`).